NSS-Wacky-Eels/WorkforceManagement
# Af view dep

## Link to Ticket
Closes # [Link](https://github.com/NSS-Wacky-Eels/WorkforceManagement/issues/6)

## Description of Proposed Changes
- User will now see, under the departments IndexPage, each department; including, name, budget, and the total number of employees for each department. 
-Test Integration for Get_IndexReturnsSuccessAndCorrectContentType will verify that the data for the first department exists on the Index page for Departments.  

## Steps to Test
- Go to the index page for departments and make sure that you can see the name, budget, and number of employees for each department. 
-Run integration tests and verify that Get_IndexReturnsSuccessAndCorrectContentType succeeds. 

```
git fetch --all
git checkout <af-viewDep>
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* DepartmentController
* Department Model
* Department Tests
* DepartmentViewModel

## Mentions @username

@KaylaReid
@alexfont321
@thatmikeparrish
@taylorlgulley


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
